.github/workflows/build-test.yml

```diff
@@ -103,7 +103,7 @@ jobs:
         strategy:
             max-parallel: 1
             matrix:
-                image: [ 'nodejs-centos6-default', 'nodejs-centos7-node12' ]
+                image: [ 'nodejs-centos6-default', 'nodejs-centos7-node12', 'nodejs-centos7-fips' ]
```

Comment:
do we need to run all those linux tests as part of required job?

Reply:
@sfc-gh-kdama Please check out the comment above

Reply:
Yeah possibly we need jenkins job changes as well.  But i leave that call up to you @sfc-gh-dprzybysz .

Reply:
Now jobs are parallel so it's not a big problem, but those builds are doing nearly the same